current-work/README.md

```diff
@@ -0,0 +1,48 @@
+# The JSON Schema Organization's areas of focus or interest
+
+## About
+### What is this space?
+
+During an [Open Community Working Meeting](https://github.com/json-schema-org/community/issues/244), it was suggested that we could have a place for the core team (and maybe active community members?) to log their current areas of focus or interest in the form of GitHub Issues, Pull Requests, and Discussions.
+
+### Why?
+
+We concluded there was a LOT of activity taking place, and it was hard to get people to focus on the "important things". It is indeed hard to focus on priority items when there are many new GitHub Issues and Discussions each week, especially for those who are community members or not working on JSON Schema full or even part time. It was suggested that by asking for one to three items of priority, we might find it easier to draw attention where we feel it is needed.
+
+### How?
+
+The process will initially be manual and require people to commit to the adjacent file which contains the list.
+The list will be used as it is found on Friday to help assist populate the following Monday's Open Community Working Meeting agenda.
+We may introduce automations to assist in this at a later point, or a medium that isn't storing data in a text file.
+
+#### How should the file be updated?
+
+Initially, it's expected that only those who are actively contributing should look at or add items to the list.
+To add items to the list, either make commits directly in this repo.
```

Comment:
Remove "either".  We should probably emphasize that pushing commits to this repo for any other file remains (forbidden? discouraged?  subject to process X? idk what the right statement is).